Frameworks/include/CGGradientInternal.h

```diff
@@ -17,8 +17,8 @@
 
 #pragma once
 
-#include "CoreGraphics/CGGradient.h"
-#include "CoreGraphicsInternal.h"
+#import <CoreGraphics/CGGradient.h>
+#import "CoreGraphicsInternal.h"
 #include <objc/runtime.h>
 
 class __CGGradient : private objc_object {
```

Comment:
i'm not implementing CGGradient (it's already implemented in a poor way).
I'm merely reading the data. It needs to be properly implemented (separate CR #1541)

---
In reply to: [92517645](https://github.com/Microsoft/WinObjC/pull/1535#discussion_r92517645) [](ancestors = 92517645)